php_sql/electives.php

```diff
@@ -0,0 +1,69 @@
+<?php
+define("MAX_TITLE_LENGTH", 128, true);
+define("MAX_TEACHER_LENGTH", 1024, true);
+define("MIN_DESCRIPTION_LENGTH", 10, true);

```

Comment:
For the current task, due to the DB constraints also the max length of the description is limited: https://iliasky.com/www/presentations/PHP-SQL.html?read=1#26 You may want to add it so that the problem is detected earlier, on backend level and not on DB level, so that the error is properly handled and displayed to the end user.